hadoop-tools/hadoop-distcp/src/main/java/org/apache/hadoop/tools/SimpleCopyListing.java

```diff
@@ -119,6 +119,16 @@ protected SimpleCopyListing(Configuration configuration,
     this.randomizeFileListing = randomizeFileListing;
   }
 
+  protected void initSimpleCopyListing(Configuration pConfiguration,
```

Comment:
Can you please mark this as `VisibleForTesting`?

Maybe `TestCopyListing` shouldn't be extending `SimpleCopyListing`, but no need to take on further refactoring right now.

Reply:
> Can you please mark this as `VisibleForTesting`?

Thank you for your explanation! I have added the `VisibleForTesting` annotation above the `initSimpleCopyListing` function.

> Maybe TestCopyListing shouldn't be extending SimpleCopyListing, but no need to take on further refactoring right now.

This is a great suggestion! I will record it as a to-do. Once we complete the upgrade from JUnit4 to JUnit5, we will implement this optimization.